cirq_superstaq/custom_gates.py

```diff
@@ -445,7 +447,7 @@ def phi(self) -> float:
     def theta(self) -> float:
         return self.exponent * np.pi
 
-    def __pow__(self, power: float) -> "RGate":
+    def __pow__(self, power: Union[float, sympy.Expr]) -> "RGate":
```

Comment:
fine as is, but fwiw this is equivalent to
```suggestion
    def __pow__(self, power: cirq.TParamVal) -> "RGate":
```
(elsewhere as well)

Reply:
Done.